app/views/nodes/notes/_empty_state_actions.html.erb

```diff
@@ -0,0 +1,7 @@
+<p>
+  Use the 
+  <%= link_to 'javascript:void(0)', data: { behavior: 'notes-dropdown-toggle' } do -%>
```

Comment:
Can't this partial and the one that's exactly the same be re-used. The only difference is `evidence-dropdown-toggle` vs `notes-dropdown-toggle`. I've pretty sure we can pull a type or resource type or something from somwhere.